casepro/backend/junebug.py

```diff
@@ -40,6 +46,73 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **params):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % self.base_url
+
+        identities = self.get_paginated_response(
+            url, params=params)
+
+        # Users who opt to be forgotten from the system have their details
+        # stored as 'removed'
+        return (
+            IdentityStoreContact(i) for i in identities if
+            i.get('details').get('name') is not "removed"
```

Comment:
I've just realised a great big problem with this anyway. Because of this filter we never actually return any identities that have been deleted :facepalm: